### DISCUSSION TITLE
I can't import any module inside nuxt3

### DISCUSSION THREAD

**Discussion by LeonardoMussato** [nuxt/nuxt#18332] [Questions] [answered]
Hi, 
I'm trying to import some modules inside my nuxt app (for example @nuxt/image) but it's not working.

If I add the module to buildModules it's giving me an error version as "Invalid Version: undefined"
If I add the module to Modules it's giving me an error on options as "Options is not defined"

Could you please help me?
It's very annoying!

**Comment by misaon** [comment, marked as answer]
@LeonardoMussato look here https://github.com/nuxt/framework/discussions/751

**Reply by LeonardoMussato** [reply]
Thank you!
